.github/workflows/test-deploy-docs.yml

```diff
@@ -0,0 +1,23 @@
+name: Test deployment
+
+on:
+  pull_request:
+    branches: [lt-docusarus]
+    paths: [docs/**]
+
+jobs:
+  test-deploy:
+    name: Test deployment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+          cache: npm
+          cache-dependency-path: ./docs/package-lock.json
+      - name: Test build
+        working-directory: docs
+        run: |
+          npm ci
+          npm run build
```

Comment:
I don't think we need a test deployment.

Reply:
Fair